Features/Users/users.feature

```diff
@@ -67,3 +67,10 @@ Feature: Use the eZ Users field
         And I fill in "Password" with "12345"
         And I fill in "Confirm password" with "123456"
         Then I should see error messages
+
+    @javascript @edge
+    Scenario: Validate that users have the "Delete" button available
+        Given I am on the Users page
+        And there is a User with name "One"
+        When I go to User "One" page
+        Then I should see a "Delete" button in the action bar
```

Comment:
We may want to complete that: what we are testing here is indeed that we are getting a delete button on users, but _also_ that we are _not_ getting a "send to trash" button.

What about

```
Then I should not see a "Send to trash" button in the action bar
 But I should see a "Delete" button in the action bar
```

(what about changing the above with `I should see a "Delete" action bar button` ? Less clutter :-)